### PR TITLE
Update Mfi switch component configuration variable

### PR DESCRIPTION
**Pull request by klaasnicolaas**
**Description:**
Update style of Mfi switch component documentation to follow new configuration variables description.
Related to #6385.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [ ] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/switch.mfi.markdown [hunk 1]
```diff
@@ -28,11 +28,32 @@ switch:
...
+username:
+  description: The mFi admin username.
+  required: true
+  type: string
+password:
+  description: The mFi admin password.
+  required: true
+  type: string
+ssl:
+  description: If `True`, use SSL/TLS to contact the mFi controller.
```
Comment (tjorim) [inline source/_components/switch.mfi.markdown]: true should be lowercase

File: source/_components/switch.mfi.markdown [hunk 2]
```diff
@@ -28,11 +28,32 @@ switch:
...
+  required: false
+  default: true
+  type: boolean
+verify_ssl:
+  description: Set this to `False` if your mFi controller has a self-signed certificate.
```
Comment (tjorim) [inline source/_components/switch.mfi.markdown]: false should be lowercase

**Review by tjorim — CHANGES_REQUESTED** [review]
You're on a roll!

File: source/_components/switch.mfi.markdown (hunk 1 above)
Comment (klaasnicolaas) [reply source/_components/switch.mfi.markdown]: I do not think that is checked so strictly, as long as the default variables are lowercase but will adjust it.

**Review by fabaff — APPROVED** [review]
Thanks :bird:

**Pull request merged**